admin/tool/policy/classes/api.php

```diff
@@ -162,6 +162,28 @@ public static function get_policy_version_status($policy, $versionid) {
         return $status;
     }
 
+    /**
+     * Returns a list of all current policy versions id.
+     *
+     * @return array list of policy version ids.
+     */
+    public static function get_current_policy_versionids() {
```

Comment:
So this is a new method that does not seem to be used anywhere in the patch? Also, how useful is it to know just the ids? Why not load everything you need in one SQL query?

Reply:
Sorry, I forgot to remove it. It was previous to change the API for creating the get_user_acceptances method. I'll remove in the following commit. Thanks for reviewing! :-)